src/Workspaces/Core/Portable/PublicAPI.Unshipped.txt

```diff
@@ -8,6 +8,7 @@ Microsoft.CodeAnalysis.Editing.DeclarationModifiers.IsFile.get -> bool
 Microsoft.CodeAnalysis.Editing.DeclarationModifiers.WithIsFile(bool isFile) -> Microsoft.CodeAnalysis.Editing.DeclarationModifiers
 Microsoft.CodeAnalysis.Editing.SyntaxEditor.SyntaxEditor(Microsoft.CodeAnalysis.SyntaxNode root, Microsoft.CodeAnalysis.Host.HostWorkspaceServices services) -> void
 *REMOVED*static Microsoft.CodeAnalysis.Editing.SyntaxGenerator.DefaultRemoveOptions -> Microsoft.CodeAnalysis.SyntaxRemoveOptions
+Microsoft.CodeAnalysis.Solution.Services.get -> Microsoft.CodeAnalysis.Host.HostWorkspaceServices
```

Comment:
So this seems a bit strange since HostWorkspaceServices still has a Workspace property; is the intent for that to go away or do we need a different type here?

Reply:
Note my concern here is generally mitigated if we keep this property internal while we sort out this question. Happy to sign off on that.